### PR TITLE
Allow content to be io.Reader

### PR DESCRIPTION
**Pull request by icholy**

### CODE REVIEW

File: pop.go [hunk 1]
```diff
@@ -107,16 +109,25 @@ func generateFile(root string, name string, content interface{}) error {
 	defer f.Close()
 
 	// Generate the content only if it is non-nil or a non-empty string
```
Comment (aseure) [inline pop.go]: Could you also update this comment?

**Review by aseure — CHANGES_REQUESTED** [review]
Good idea, thank you for the contribution.

**Comment by aseure** [comment]
Could you also update the `README.md` to explicitly say that `io.Reader` is supported?

**Comment by icholy** [comment]
done

**Review by aseure — APPROVED** [review]
LGTM

**Pull request merged**